srcpkgs/webrtc-audio-processing/template

```diff
@@ -13,7 +13,7 @@ distfiles="${FREEDESKTOP_SITE}/pulseaudio/webrtc-audio-processing/webrtc-audio-p
 checksum=95552fc17faa0202133707bbb3727e8c2cf64d4266fe31bfdb2298d769c1db75
 
 # Upstream issue: https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/issues/5
-if [ "$XBPS_MACHINE" = "i686" ]; then
+if [ "$XBPS_MACHINE" = "i686" ] || [ "$XBPS_MACHINE" = "i686-musl" ]; then
```

Comment:
```suggestion
if [[ "$XBPS_TARGET_MACHINE" = "i686"* ]]; then
```

Reply:
In addition to classabbyamp's suggestion, please fix the commit message to follow the convention. The message used at the PR's title will suffice, though you could amend it to just say "for i686" since it seems to apply to both i686/i686-musl.

Reply:
Will do.